onnxruntime/contrib_ops/cpu/bert/group_query_attention.cc

```diff
@@ -43,6 +49,13 @@ GroupQueryAttention<T>::GroupQueryAttention(const OpKernelInfo& info)
 
 template <typename T>
 Status GroupQueryAttention<T>::Compute(OpKernelContext* context) const {
+
+  const std::string node_name = this->Node().Name();
+
+  // Initialize the profiler_ with a unique log file based on the node name
+  profiler_ = new onnxruntime::profiling::Profiler();
```

Comment:
This is test code?

Reply:
it is used to profile GQA. I have removed it because we need the code to be in for the release.